scripts/secrets/remove-old-secret-revisions.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -eu
+
+secrets=$(juju secrets --format=json | jq -r "to_entries | .[] | select( .value | .revision > 1) | .key" | sort -u)
+for secret in $secrets; do
+    owner=$(juju show-secret $secret --format=json | jq -r ".[] | .owner")
```

Comment:
Im guessing this doesn't do the right thing for non application secrets (eg user secrets)
I dont know if we care, as we don't really get this bug there. maybe we should just skip them?